bots/bugbuster/bot.definition.ts

```diff
@@ -0,0 +1,36 @@
+import * as sdk from '@botpress/sdk'
+import github from './bp_modules/github'
+import slack from './bp_modules/slack'
+
+export default new sdk.BotDefinition({
+  states: {
+    listeners: {
+      type: 'bot',
+      schema: sdk.z.object({
+        conversationIds: sdk.z.array(sdk.z.string()),
+      }),
+    },
+  },
+  events: {
+    syncIssuesRequest: {
+      schema: sdk.z.object({}),
+    },
+  },
+  recurringEvents: {
+    fetchIssues: {
+      type: 'syncIssuesRequest',
+      payload: {},
+      schedule: { cron: '0 0/6 * * *' }, // every 6 hours
+    },
+  },
+})
+  .add(github, {
+    enabled: true,
+    configurationType: null,
+    configuration: { owner: 'botpress', repo: 'botpress', token: '$TOKEN' },
```

Comment:
Good question. 

Currently, the bots of this repository are never deployed. I even deleted the `bugbuster` workflow in a previous commit. So this value is never replaced. 

My intention is to address this in a dedicated PR afterward. A quick an dirty way of doing this would be to use environment variables like `token: process.env.TOKEN!` and provide the variables when running `bp build`, `bp deploy` and `bp dev` on the bots, but I have something better in mind.